docker-compose.yml

```diff
@@ -149,3 +149,12 @@ services:
       - XMS=256M
       - XMX=1G
     restart: always
+
+  mapstore:
+    image: geosolutionsit/mapstore2-georchestra:latest
```

Comment:
Let's wait to see how https://github.com/georchestra/mapstore2-georchestra/runs/1164868350 behaves so we have a `georchestra/mapstore` image here

Reply:
@fvanderbiest  job failed with:

denied: requested access to the resource is denied

Reply:
Strange indeed. Yet I added the credentials as secrets in the repo.
Asking advice to @pmauduit

Reply:
> Strange indeed. Yet I added the credentials as secrets in the repo.
> Asking advice to @pmauduit 

Maybe the account used here for docker-hub interactions has no rights to create a new repository (and georchestra/mapstore does not exist yet on docker-hub)

Reply:
Good catch !
Created https://hub.docker.com/repository/docker/georchestra/mapstore

Reply:
Did not help ...

Reply:
looking [here](https://github.com/Azure/docker-login#you-can-build-and-push-container-registry-by-using-the-following-example
) only thing which comes to mind is to explicit the docker registry:
```
    - name: "Pushing latest to docker.io"
      if: github.ref == 'refs/heads/master' && github.repository == 'georchestra/mapstore2-georchestra'
      run: |
        docker tag georchestra/mapstore:${{ steps.version.outputs.VERSION }} index.docker.io/georchestra/mapstore:latest
        docker push index.docker.io/georchestra/mapstore:latest
```

Reply:
Yes, you should give it a try

Reply:
@fvanderbiest the management of your docker hub is not a matter of our competence in my opinion (furthermore, the workflow [here](https://github.com/georchestra/mapstore2-georchestra/blob/master/.github/workflows/mapstore.yml) was originally created by @landryb and the problem in general is not dependent by this PR as far as I can understand). We have provided our support for this too by making [this PR](https://github.com/georchestra/mapstore2-georchestra/pull/279), but I think it is something up to you now. We will obviously update [this line](https://github.com/georchestra/docker/pull/69/files#diff-4e5e90c6228fd48698d074241c2ba760R154) as soon as the problem in the docker hub is solved. I'm looking forward to hearing from you about this, thank you.

Reply:
There was a permissions problem on our side, indeed. Sorry about that.

It's now fixed and we're very happy to have a `georchestra/mapstore:latest` image now at https://hub.docker.com/repository/docker/georchestra/mapstore !